src/shaders/stdosl.h

```diff
@@ -165,6 +165,20 @@ color  mix (color  x, color  y, color  a) BUILTIN;
 color  mix (color  x, color  y, float  a) BUILTIN;
 float  mix (float  x, float  y, float  a) BUILTIN;
 #endif
+normal select (normal cond, normal x, normal y) BUILTIN;
```

Comment:
Oops, I dithered on the argument order a few times, changing this back and forth. In the end, I thought it made the most sense to be in the same order as mix(), but I may not have fixed it everywhere.